.github/workflows/CI.yaml

```diff
@@ -0,0 +1,258 @@
+
+          vagrant scp :/home/vagrant/\$i-extract/zcash-*/bin/zcash-tx ./\$i-extract/
+          vagrant scp :/home/vagrant/\$i-extract/zcash-*/bin/zcash-fetch-params ./\$i-extract/
+          vagrant scp :/home/vagrant/\$i-extract/zcash-*/bin/zcashd ./\$i-extract/
+          vagrant scp :/home/vagrant/\$i-extract/zcash-*/bin/zcash-cli ./\$i-extract/
+          vagrant scp :/home/vagrant/\$i-extract/zcash-*/bin/zcashd-wallet-tool ./\$i-extract/
+          docker cp ./\$i-extract \$i:/home/vagrant/\$i-deb-build/
+          docker exec -w /home/vagrant/\$i-deb-build \$i bash -c "rm -rf src && mv \$i-extract src && ./zcutil/build-debian-package.sh"
+          docker cp \$i:/tmp/zcbuild ./debs/\$i
+        done
+        vagrant scp :/home/vagrant/zcash-binaries ./
+        for i in \$OS;
+        do
+          cd ./zcash-binaries/\$VERSION/\$i
+          for j in \$(ls *linux64.tar.gz); do
+            mv \$j \$(echo \$j | sed 's/.tar.gz/-debian-'\$i'.tar.gz/g')
+          done
+          for j in \$(ls *debug.tar.gz); do
+            mv \$j \$(echo \$j | sed 's/.tar.gz/-debian-'\$i'.tar.gz/g')
+          done
+          gpg -u sysadmin@z.cash --armor --digest-algo SHA256 --detach-sign *debug-debian-\$i.tar.gz
+          gpg -u sysadmin@z.cash --armor --digest-algo SHA256 --detach-sign *linux64-debian-\$i.tar.gz
+          cd \$current_dir
+        done
+        export final_version=\$(cat assert2.txt | awk '{print \$2}' | grep "desc.yml" | head -n 1 | sed 's/-desc.yml//g')
+        gsutil -m rsync -r ./debs gs://${{ secrets.GCP_PROJECT_ID_PROD }}-apt-packages/debs
+        gsutil -m rsync -r ./zcash-binaries gs://${{ secrets.GCP_PROJECT_ID_PROD }}-apt-packages/zcash-binaries
+        apt install aptly -y
+
+        # generate apt
+        mkdir aptserver
+        cd aptserver
+        gsutil -m cp -r gs://${{ secrets.GCP_PROJECT_ID_PROD }}-apt-server/pool/main/z/zcash/ .
+        cd zcash
+        cp -a ../../debs/buster/zcbuild/*.deb \$final_version-amd64-buster.deb
+        cp -a ../../debs/bullseye/zcbuild/*.deb \$final_version-amd64-bullseye.deb
+        cp -a ../../debs/bookworm/zcbuild/*.deb \$final_version-amd64-bookworm.deb
+
+        ls \$final_version-amd64-buster.deb || exit 1
+        ls \$final_version-amd64-bullseye.deb || exit 1
+        ls \$final_version-amd64-bookworm.deb || exit 1
+
+        aptly repo create --distribution buster --comment "" --component main zcash_buster_amd64_repo
+        aptly repo create --distribution bullseye --comment "" --component main zcash_bullseye_amd64_repo
+        aptly repo create --distribution bookworm --comment "" --component main zcash_bookworm_amd64_repo
+        aptly repo create --distribution stretch --comment "" --component main zcash_stretch_amd64_repo
```

Comment:
Should the stretch repo still be referenced?

Reply:
Correct,  this is for the historical packages